ocw_oer_export/constants.py

```diff
@@ -1,4 +1,7 @@
 """
 Module containing constants.
 """
-API_URL = "https://mitopen.odl.mit.edu/api/v1/courses/?platform=ocw"
+API_BASE_URLS = {
+    "PRODUCTION": "https://mitopen.odl.mit.edu/api/v1/courses/?platform=ocw",
+    "RC": "https://mitopen-rc-2ea15531374d.herokuapp.com/api/v1/courses/?platform=ocw",
+}
```

Comment:
**Suggestion:** IMO, just make the env variable the base URL:

```
API_BASE_URL=https://mitopen.odl.mit.edu
# or
API_BASE_URL=https://mitopen-rc.odl.mit.edu
# or
API_BASE_URL=http://localhost:8063
```

and the library can add `api/v1/courses/?platform=ocw`

That way you can set the base url to whatever you want—local, rc, prod.